src/utils/polyfills.ts

```diff
@@ -1,6 +1,5 @@
-import 'core-js/modules/es7.object.values';
 import 'core-js/modules/es7.object.entries';
-import 'reflect-metadata';
+import 'core-js/modules/es7.object.values';
```

Comment:
why is this line after now?

Reply:
automatic import ordering.

Reply:
it shouldn't reorder these raw imports....